### PR TITLE
Converted console_scripts to scripts

### LINKED ISSUE #831: Add a main function to all our console_scripts so they are more easily runnable
https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html

We simply need to add:
```Python
if __name__ == "__main__":
    main()
```
at the end of every script.

### PR DESCRIPTION
**Pull request by RishiMagudia**
Fixes #831.

### CODE REVIEW

**Comment by marco-c** [comment]
Linting is failing, you can use `pre-commit` for that.

File: scripts/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (marco-c) [inline scripts/__init__.py]: This is not a script...

**Comment by marco-c** [comment]
You'll also need to update `setup.py` (following the docs linked in #831).

**Comment by marco-c** [comment]
@RishiMagudia why did you close the PR?

**Comment by RishiMagudia** [comment]
Gonna figure out how to actually test locally before I submit another pull

**Comment by marco-c** [comment]
You don't need to submit another PR, you can just update this one :)

**Comment by RishiMagudia** [comment]
Is their a way I can test locally without having to push every time?
My IDE did not highlight the missing comma.

**Comment by Monkatron95** [comment]
Linting is failing again, pre-commit.

**Comment by marco-c** [comment]
@RishiMagudia see above :P 
https://github.com/mozilla/bugbug/pull/837#issuecomment-517656958 :)

File: setup.py [hunk 2]
```diff
@@ -50,6 +50,8 @@ def read_requirements(file_):
             "bugbug-microannotate-generate = scripts.microannotate_generator:main",
             "bugbug-classify-commit = scripts.commit_classifier:main",
             "bugbug-regressor-finder = scripts.regressor_finder:main",
+            "bugbug-generate-sheet = scripts.generate_sheet:main",
+            "bugbug-retrieve-training-metrics = scripts.retrieve_training_metrics:main",
         ]
```
Comment (marco-c) [inline setup.py]: Again, https://github.com/mozilla/bugbug/pull/837#issuecomment-517657249 :P
These should no longer be console_scripts, but scripts.

You'll also need to rebase on top of master, as a few more scripts have been added recently.

**Comment by marco-c** [comment]
You'll also need to add `#!/usr/bin/env python` at the beginning of all script files.

**Comment by marco-c** [comment]
There's something wrong in the history of your PR, maybe a merge gone wrong.

**Comment by RishiMagudia** [comment]
My git-fu is weak. 
I think my git issue stemmed from not creating a branch off master in my repo.

I did the following commands to get where I am

`git remote add bugbug https://github.com/mozilla/bugbug`
`git fetch bugbug`
`git rebase -i bugbug`

I should have rebased onto master and then in a separate branch, rebased off that.

My current thoughts are to just copy my changes and create a new repo with the latest of mozilla/bugbug and then create a new pull request as I have no experience in git history mangling and this commit history is extremely messy.

I appreciate the help and the patience and thoughts on the above would be appreciated.

**Comment by marco-c** [comment]
Just copy your changes somewhere, then run:
```
git pull upstream master
git reset --hard upstream/master
```
Then copy back your changes, add them and commit.
Then, when you need to push, do `git push origin master --force`.

N.B.: If you don't have 'upstream', run
```
git remote add upstream git@github.com:mozilla/bugbug.git
```
first.

**Comment by marco-c** [comment]
Closing as the PR is stale. Feel free to reopen if you still intend to work on this.

**Pull request closed without merging**